src/lib/backend_interface.js

```diff
@@ -1,9 +1,83 @@
 class BackendInterface {
+  // DROP SITES
```

Comment:
I have a small heart attack every time I see this because my brain reads it as a SQL statement. :)